src/qtscriptdebug.cpp

```diff
@@ -145,6 +145,7 @@ static void fillViewdataModel(QStandardItemModel &m)
 	for (const WzString& key : keys)
 	{
 		VIEWDATA *ptr = getViewData(key);
+		ASSERT(ptr->type < view_type.size(), "Bad viewdata type");
```

Comment:
Doesn't "PPL" a few lines above need to be "RPL" also?

Reply:
Yes, fixed.